airsonic-main/src/main/java/org/airsonic/player/controller/MainController.java

```diff
@@ -134,6 +134,7 @@ protected ModelAndView handleRequestInternal(@RequestParam(name = "showAll", req
             map.put("sieblingAlbums", sieblingAlbums);
             map.put("artist", guessArtist(children));
             map.put("album", guessAlbum(children));
+            map.put("mbReleaseId", guessMbReleaseId(children));
```

Comment:
Maybe something like musicBrainzId would be better for naming of this element?

Reply:
Well we have to be careful because there could also be Music Brainz Track ID, Music Brainz Artist ID, Music Brainz Disc ID, etc. So I think we need 'releaseId' at a minimum. `mbReleaseId` or `musicBrainzReleaseId`. Best thing would probably be `musicBrainzReleaseId` in the code base and `mb_release_id` for the database column name.

Reply:
`musicBrainzReleaseId' sounds good!